monai/networks/nets/resnet.py

```diff
@@ -207,18 +199,31 @@ def __init__(
         ]
 
         block_avgpool = get_avgpool()
-        conv1_kernel, conv1_stride, conv1_padding = get_conv1(conv1_t_size, conv1_t_stride)
         block_inplanes = [int(x * widen_factor) for x in block_inplanes]
 
         self.in_planes = block_inplanes[0]
         self.no_max_pool = no_max_pool
 
+        if isinstance(conv1_t_size, int):
```

Comment:
this input validation could be done with https://github.com/Project-MONAI/MONAI/blob/d8121d235178c8a8b123a71f4cd53b23e79dac2d/monai/utils/misc.py#L113-L133